src/app/page.tsx

```diff
@@ -1,23 +1,47 @@
-import { PetGrid } from '@/components/PetGrid';
-import { AddPetModal } from '@/components/modals/AddPetModal';
-import { Toaster } from '@/components/ui/toaster';
 import { font_accent } from '@/components/ui/fonts';
 import Link from 'next/link';
 import { Button } from '@/components/ui/button';
 import '@/app/styles/main.scss';
 
 export default function Home() {
   return (
-    <div className="grid place-items-center h-[100vh] text-center doors-of-durin">
+    <div className="h-[100vh] w-full home">
       <main className="text-center">
-        <h1 className={`${font_accent.className} font-semibold text-3xl`}>
-          Pet app home page
-        </h1>
-        <div>
-          <p>Speak, friend, and</p>
-          <Link href="user">
-            <Button>Enter</Button>
-          </Link>
+        <div className="home-img home-content">
+          <div className="home-img__blob home-img__blob--first"></div>
+          <div className="home-img__blob home-img__blob--second"></div>
+          <div className="home-img__blob home-img__blob--third"></div>
+          <div className="home-img__blob home-img__blob--fourth"></div>
+        </div>
+
+        <div className="home-text home-content">
+          <h1
+            className={`${font_accent.className} font-semibold text-4xl text-white-100`}
+          >
+            Welcome to Pet App
+          </h1>
+          <h2 className="text-xl text-white-100 text-balance">
+            Keep Your Pet's Health in Check, One Paw at a Time!
+          </h2>
+          <h3 className="text-xl text-white-100 text-balance">
+            Track their medical history, manage appointments, and ensure a
+            healthy, happy life for your furry friend.
+          </h3>
+          <div className="flex gap-4">
+            <Link href="user">
+              <Button
+                variant={'outline'}
+                className="text-white-100 hover:text-primary hover:bg-white-100"
+              >
+                Login
+              </Button>
+            </Link>
+            <Link href="user">
+              <Button className="bg-white-100 text-primary hover:bg-transparent hover:text-white-100 border hover:border-white-100">
+                Signup
+              </Button>
+            </Link>
```

Comment:
Regarding buttons. At the moment in our app we have three types of buttons:
- default (bg-primary, white text)

![image](https://github.com/WitchDevelops/cardboard-fort/assets/112077394/e6569898-43c9-4f17-9eaa-7add7c5829b4)

- outline (bg-transparent, white text)

![image](https://github.com/WitchDevelops/cardboard-fort/assets/112077394/d0c01fbb-067b-4136-bd4b-b059db2acb75)

- all white (bg-white, text primary)

![image](https://github.com/WitchDevelops/cardboard-fort/assets/112077394/4ebcd83a-7f3d-4de5-a0fe-4cb9e69f63a2)

To keep the code scalable, let's not style each button individually, but use button variants instead. 
Here `src\components\ui\button.tsx` we have: `default`, `desctructive`, `outline`, `secondary`, `ghost`, and `link` button variants at our disposal. As `secondary` I would keep the one with bg-secondary (which is currently green). Let's add another button variant (`white`? or something like that), style it here, and use on the home page. 
Or, since the `Login` and `Sign up` buttons are basically the same button variant (by that I mean that the `Sign up` button sort of has the `hover` state style of the outline button as a default, and goes to `outline` variant on hover), we can use `outline` on both? or add a `outline-reverse` button variant...?

Point is, let's use button variants instead of inline styles, and customize those variants to suit our needs.